kubeflow/trainer/rhai/traininghub.py

```
@@ -491,6 +491,55 @@ def log_message(self, format, *args):
+            files_removed = 0
+            for pattern in patterns:
+                full_pattern = os.path.join(ckpt_output_dir, pattern)
+                for file_path in glob.glob(full_pattern):
+                    try:
+                        os.remove(file_path)
+                        files_removed += 1
+                        print(
+                            f"[Kubeflow] Removed stale metrics file: {os.path.basename(file_path)}",
+                            flush=True,
+                        )
+                    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)